backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -199,16 +203,33 @@ public LiveOktaRepository(
         oktaGroupList.stream()
             .filter(g -> orgDefaultGroupName.equals(g.getProfile().getName()))
             .findFirst()
-            .orElseThrow(
-                () ->
-                    new IllegalGraphqlArgumentException(
-                        "Okta group not found for this organization"));
+            .orElseThrow(() -> new IllegalGraphqlArgumentException(OKTA_GROUP_NOT_FOUND));
 
     return orgDefaultOktaGroup.listUsers().stream()
         .map(u -> u.getProfile().getEmail())
         .collect(Collectors.toUnmodifiableSet());
   }
 
+  public ApiOrganizationAdmin getAdminUserForPendingOrganization(Organization org) {
+    final String orgDefaultGroupName =
+        generateRoleGroupName(org.getExternalId(), OrganizationRole.getDefault());
+    final GroupList oktaGroupList =
+        _client.listGroups(orgDefaultGroupName, FILTER_TYPE_EQ_OKTA_GROUP, null);
+
+    Group orgDefaultOktaGroup =
+        oktaGroupList.stream()
+            .filter(g -> orgDefaultGroupName.equals(g.getProfile().getName()))
+            .findFirst()
+            .orElseThrow(() -> new IllegalGraphqlArgumentException(OKTA_GROUP_NOT_FOUND));
+
+    User adminUser = orgDefaultOktaGroup.listUsers().single();
```

Comment:
You may want to consider making this operation safe even if there are multiple admins in the group. It should be nearly impossible for there to be multiple admins at this stage, unless someone manually manipulated the Okta groups, but I made a very similar change a couple weeks ago (fetching admin email in the case of duplicate orgs for the customized errors) that wasn't gated to allow multiple admin users and immediately got paged for it. (Okta throws a nasty exception if you call `single` on a group with multiple users)

This was the fix: https://github.com/CDCgov/prime-simplereport/pull/2652